modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+//provides includes for reference sections in anypoint-insights.adoc 
+
+//-----------------------//
+// APP DETAIL HEADER INFO:
+//tag::app-detail-header[]
+Type::
+Mule application type (*Application*)
+
+Status::
+Status code that indicates whether this Mule app is running or in some other xref:runtime-manager::managing-deployed-applications.adoc#status_states[state]
++
+This code comes from Runtime Manager.
+
+URL:: 
+URL of this deployed Mule app
+
+//TODO: Runtime -> Runtime Version (SATHISH)
+Runtime::
+Mule runtime engine version of this Mule app
+
+Replicas::
+Virtual core fraction and count for this Mule app, for example, *0.1 vCores x 1* (= 10% of 1 vCore)
+
+Last Updated::
+Date-time stamp for the last redeployment of this Mule app
```

Comment:
I wonder if we can simplify this by saying `date-time at which the Mule application was last updated`. The fact that redeployment happens when the app is changed is a subtle implementation detail that the user may or may not care about.

Reply:
@sathishpaulleo We talked about this, and it turns out that any redeployment, even without any other change, counts as an update.

```suggestion
Date-time stamp for the last update or redeployment of this Mule app
```